testplan/monitor/resource.py

```diff
@@ -43,11 +44,34 @@ class ProcessResourceData(ResourceData):
     pid: int
 
 
+class HostResourceRow(NamedTuple):
```

Comment:
Posting here, but there is a lack of docstrings for new classes and methods/functions added in the PR.